pkg/cmd/gitprovider/add.go

```diff
@@ -62,3 +143,20 @@ var GitProviderAddCmd = &cobra.Command{
 		return nil
 	},
 }
+
+var aliasFlag string
+var usernameFlag string
+var baseApiUrlFlag string
+var tokenFlag string
+var signingMethodFlag string
+var signingKeyFlag string
+
+func init() {
+	GitProviderAddCmd.Flags().StringVarP(&aliasFlag, "alias", "a", "", "Alias")
+	GitProviderAddCmd.Flags().StringVarP(&usernameFlag, "username", "u", "", "Username")
+	GitProviderAddCmd.Flags().StringVarP(&baseApiUrlFlag, "baseApiUrl", "b", "", "BaseApiUrl")
```

Comment:
Should be `base-api-url`.

Also, descriptions can have spaces. e.g. `BaseApiUrl` => `Base API Url`, `Signing Method (ssh, gpg)`...

Reply:
done